examples/medical-center/medicalneed_client.bal

```diff
@@ -48,7 +48,7 @@ client class MedicalNeedClient {
     }
 
     remote function readByKey(int key) returns MedicalNeed|error {
-        return (check self.persistClient.runReadByKeyQuery(MedicalNeed, key)).cloneWithType(MedicalNeed);
+        return (check self.persistClient.runReadByKeyQuery(MedicalNeed, key, [])).cloneWithType(MedicalNeed);
```

Comment:
```suggestion
        return (check self.persistClient.runReadByKeyQuery(MedicalNeed, key)).cloneWithType(MedicalNeed);
```

Reply:
without passing an empty array should also work because it is a defaultable param

Reply:
Fixed in [2122a98](https://github.com/ballerina-platform/module-ballerina-persist/pull/11/commits/2122a98b0b088688e1d52b857bb174e574673687)